packages/calcite-components/src/components/action/action.scss

```diff
@@ -227,3 +223,19 @@
 }
 
 @include base-component();
+@include disabled {
+  //   :host([disabled]) .button,
```

Comment:
question: Can we remove these comments?

Reply:
question: Are we sure this CSS isn't necessary?

Reply:
just double checked. Removing this code makes no changes to the UI. I've added a test to storybook for the component in an active-disabled state.

Reply:
Thanks. sounds good!